pkgs/applications/networking/cloudflared/default.nix

```diff
@@ -1,11 +1,11 @@
 { lib
 , stdenv
-, buildGoModule
+, buildGo120Module # https://github.com/cloudflare/cloudflared/issues/1054
```

Comment:
I think preferred way, is to override in all-packages.nix

this is also preferred way post migration to by-name package location https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
Fixed!